rs/tests/nested/src/util.rs

```diff
@@ -80,24 +80,6 @@ pub fn setup_ic_infrastructure(env: &TestEnv, dkg_interval: Option<u64>) {
         .expect("failed to setup ic-gateway");
 }
 
-/// Asserts that SetupOS and initial NNS GuestOS image versions match.
-/// Only checks if both functions return ReplicaVersion successfully.
-/// NOTE: If you want to create a new test with conflicting versions, add a
-/// field to override this check and, in your test, account for the fact that
-/// after registration, the deployed node will upgrade to the NNS GuestOS version.
-pub fn assert_version_compatibility() {
-    let setupos_version = get_setupos_img_version();
-    let guestos_version = get_guestos_img_version();
-
-    if setupos_version != guestos_version {
-        // TODO: Revert change after extending image version support
-
-        // panic!(
```

Comment:
Do we want some log, `"Initial GuestOS and SetupOS versions do not match, are you using 'without_unassigned_config()'?"`?